client/src/components/pages/LogIn/index.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import singUpValidation from '../utils/schema';
+import Input from '../../common/Input';
+import Button from '../../common/Button';
+
+import('./index.css');
+
+class LogIn extends Component {
+  state = {
+    username: '',
+    password: '',
+    error: false,
+  };
+
+  onChangHandler = ({ target: { name, value } }) => {
+    this.setState({
+      [name]: value,
+    });
+  };
+
+  handleSubmit = e => {
+    e.preventDefault();
+    const { username, password } = this.state;
+    singUpValidation.isValid({ username, password }).then(res => {
+      if (!res) {
```

Comment:
you can remove this if-else completely and do just `this.setState({ error: !!res });` (the `!!` will evaluate res as boolean)

Reply:
the isValid promise return a boolean value 
so what you think ?

Reply:
yeah it should be fine then without the `!!`. To make it more descriptive, you can rename `res` to `isValid`. Also the `error` should be called `isError` to make it clear it's boolean and not an actual error/error msg